app/google-services.json

```diff
@@ -1,36 +0,0 @@
-// Follow the steps in https://developers.google.com/mobile/add to generate your own google-services.json file.
```

Comment:
Does the project compile without this placeholder json file?

Reply:
no, it doesn't.. is it so bad if it fails to build if its missing a firebase config? I think it was brought in via a merge or something...

Reply:
We added that place holder json file so that the project compiles and works for outgoing calls. A real json is needed only if the app wants to receive incoming calls.

Reply:
I can add it back if you think its better

Reply:
Let's add that back.